best-practices/README.md

```diff
@@ -322,6 +322,7 @@ Ember
 * Prefer dependency injection through `Ember.inject` over initializers, globals
   on window, or namespaces. ([sample][inject])
 * Prefer sub-routes over maintaining state.
+* Prefer explicit setting of boolean properties over `toggleProperty`.
```

Comment:
From my understanding of the explanations, this is a better option and you should watch out for the alternative. There are cases that using `toggleProperty` is a better option but not the normal case.